core/domain/email_services.py

```diff
@@ -17,10 +17,19 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+import logging
+import re
+from textwrap import dedent # pylint: disable=import-only-modules
```

Comment:
In this codebase, Python stdlibs are imported at the module layer. So please just do `import textwrap`.

Also, if you think you need to use a pylint pragma -- heed the warning and don't use it. In general, if you add a pylint-disable the reviewer will ask you to remove it. The lint checks in the codebase should be adhered to by default.